### ISSUE TITLE
dot record console.log('xx')

### ISSUE THREAD

**Issue by junbao520** [log4js-node/log4js-node#1042]
log4js.configure({
  appenders: {
    cheeseLogs: { type: 'file', filename: 'cheese.log' },
    console: { type: 'console' }
  },
  categories: {
    cheese: { appenders: ['cheeseLogs'], level: 'error' },
    another: { appenders: ['console'], level: 'trace' },
    default: { appenders: ['console', 'cheeseLogs'], level: 'trace' }
  }
});

this code  dot record console.log()

**Comment by rnd-debug** [comment]
@junbao520 Please provide an example of your issue. 
The following code works as expected: 
```
const log4js = require('log4js')

log4js.configure({
appenders: {
cheeseLogs: { type: 'file', filename: 'cheese.log' },
console: { type: 'console' }
},
categories: {
cheese: { appenders: ['cheeseLogs'], level: 'error' },
another: { appenders: ['console'], level: 'trace' },
default: { appenders: ['console', 'cheeseLogs'], level: 'trace' }
}
});

const logger = log4js.getLogger();
logger.trace("this message is recorded to cheese.log file and appears in the console");
console.log("this message goes to console only, as it is not using log4js");
const loggerAnother = log4js.getLogger("another");
loggerAnother.trace("goes only to console");
const loggerCheese = log4js.getLogger("cheese");
loggerCheese.error("goes only to cheese file");
loggerCheese.trace("not recorded, because cheese level is 'error'");
```

**Comment by abernh** [comment]
I saw the same issue and it is a misunderstanding:

## TLDR;
`logger.log` is not a log-level-short method like `logger.info` or `logger.error` but expects a valid log-level as first parameter (or will silently default to log-level `Info`).

## In Detail
I landed here because my `logger.log` calls seem to drop the first argument that was given, logging only 2nd and further arguments.
As I had simply replaced `console.log` with the `log4js` logger, I had overlooked the fact that there is no `log` log-level. The `.log` method has a different format.

```
#lib/logger.js:70
  log(level, ...args) { ...
``` 

Using `logger.log('some message');` will not log that message but treats the log as log-level `Info` and drops silently that first argument.

```
const log4js = require('log4js')

log4js.configure({
appenders: {
console: { type: 'console' }
},
categories: {
default: { appenders: ['console'], level: 'trace' }
}
});

const logger = log4js.getLogger();
logger.trace("this message appears in the console");
logger.log("this message appears as empty info log in the console");
```
